Tests/aweXpect.Reflection.Tests/ThatType.DoesNotInheritFrom.Tests.cs

```diff
@@ -0,0 +1,39 @@
+using Xunit.Sdk;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatType
+{
+	public sealed class DoesNotInheritFrom
+	{
+		public sealed class Tests
```

Comment:
The tests for the negated case using `DoesNotComplyWith` (see [ThatFields.ArePublic.NegatedTests](https://github.com/aweXpect/aweXpect.Reflection/blob/7ca54066e02e76194e0e78a2d1dc751c13b8009e/Tests/aweXpect.Reflection.Tests/ThatFields.ArePublic.Tests.cs#L45) for an example) are missing.